frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -97,7 +97,7 @@ class DashboardHeader extends Component {
     onChangeLocation: PropTypes.func.isRequired,
 
     toggleSidebar: PropTypes.func.isRequired,
-    sidebar: PropTypes.string.isRequired,
+    sidebar: PropTypes.object.isRequired,
```

Comment:
From my testing it seems like `sidebar` always takes in:

```
{
    props: {...},
    name: <sidebar type name>,
}
```

Reply:
I have a better fix for it 😄 but great you touched it

Reply:
Oooh ok - I'd much rather use a fix from someone who knows their way around a little better, what were you planning to use here?